Sources/MongoSwift/BSON/BsonValue.swift

```diff
@@ -45,37 +45,9 @@ public enum BsonType: Int {
     /// 128-bit decimal floating point
     decimal128,
     /// Special type which compares lower than all other possible BSON element values
-    minKey,
+    minKey = 0xff,
```

Comment:
Since BsonTypeMap in its old form has been removed, would it make sense to explicitly define the enum values for all cases here? That would make it easier to casually glance at a case in this enum and immediately know what its code is, instead of needing to count forward from the initial case (i.e. `0x00`).